deploy/template.yaml

```diff
@@ -106,7 +106,7 @@ Mappings:
       GOVUKNOTIFYREMINDERTEMPLATEID: "2987ded5-1c1b-4336-931b-7f66a5569684"
       YOTISESSIONTTLDAYS: 10 # Default 10 days
       RESOURCETTLSECS: 1209600 # Default 14 days
-      CLIENTS: '[{"jwksEndpoint":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/redirect"}]'
+      CLIENTS: '[{"jwksEndpoint":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/redirect?id=f2f"}]'
```

Comment:
is the code we use to construct the redirect is tolerant of both forms here - i.e. if the `redirectUri` already contains a query parameter it appends the `error` query parameter with an `&`, whereas if there is no query parameter in the `redirectUri` it appends the `error` with a `?` ? Do we have unit tests for both scenarios?

Reply:
Yes @darrendunford [Test for no id=f2f](https://github.com/alphagov/di-ipv-cri-f2f-api/blob/d31c84c9dcd6eba29207297f54d435571d9302ec/src/tests/unit/services/AbortRequestProcessor.test.ts#L78) & [Test for id=f2f](https://github.com/alphagov/di-ipv-cri-f2f-api/blob/d31c84c9dcd6eba29207297f54d435571d9302ec/src/tests/unit/services/AbortRequestProcessor.test.ts#L90)